gulpfile.js

```diff
@@ -254,8 +235,12 @@ gulp.task('svg-sprite:build', function () {
 });
 
 gulp.task('svg2png', function () {
-  gulp.src(path.join(paths.svg, '**', '*.svg'))
-  .pipe(svg2png())
+  const scale = 1;
+  const verbose = true;
+  const concurrency = 1;
+
+  return gulp.src(path.join(paths.svg, '**', '*.svg'))
+  .pipe(svg2png(scale, verbose, concurrency))
```

Comment:
Nope, we're on 1.0.2 which has this: https://github.com/akoenig/gulp-svg2png/blob/b6b9ab215333a43cafbb8e5b870bc8c4fd0f3382/index.ts#L113